build-image-src/Dockerfile-dotnet6

```diff
@@ -33,7 +33,8 @@ FROM public.ecr.aws/sam/emulation-dotnet6:latest-$IMAGE_ARCH
 COPY --from=1 /rootfs /
 
 # Install AWS CLI
-RUN curl 'https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip' -o ./awscliv2.zip && unzip ./awscliv2.zip && ./aws/install && rm ./awscliv2.zip && rm -rf ./aws
+ARG AWS_CLI_ARCH
```

Comment:
How/where is this `ARG` passed in?

Reply:
Here https://github.com/aws/aws-sam-build-images/blob/develop/build-image-src/build_all_images.sh#L30 and here https://github.com/aws/aws-sam-build-images/blob/develop/build-image-src/build_all_images.sh#L46

Reply:
We already pass it in. Got it, was just missed in the dockerfile. :)